### ISSUE TITLE
got10k validation set GOT-10k_Val_000136 issue

### ISSUE THREAD

**Issue by jonathantompson** [got-10k/toolkit#25]
The validation set sample, GOT-10k_Val_000136 changes frame resolution between frames 00000111.jpg -> 00000112.jpg (1920 x 1080 --> 1280 x 720).

Was this intended?

**Comment by jonathantompson** [comment]
This is also present in GOT-10k_Val_000137, and about 8 sequences in the training set....
